lib/elixir/lib/module.ex

```diff
@@ -351,6 +351,10 @@ defmodule Module do
   the documentation for the [`:compile` module](http://www.erlang.org/doc/man/compile.html).
   '''
 
+  @typep function_arity :: {atom, arity}
```

Comment:
I think we can name it `definition`, as it this name is already used in `definitions_in` function, which returns in exactly that format.